routes/index.js

```diff
@@ -191,11 +191,66 @@ module.exports = function (passport) {
 
 
   router.get('/userManagement', checkAuth, function (req, res) {
-    dbAPI.getUsers().then(function (ret) {
-      res.render('userManagement.njk', { obj: ret, authenticated: true });
-    });
+
+    Promise.all([
+      dbAPI.getUsers(undefined),
+      dbAPI.getStations(true)
+
+    ])
+      .then(
+        ([allUsers, allStations]) => {
+          var data = {
+            users: allUsers,
+            stations: allStations,
+            authenticated: true
+          }
+
+          res.render('userManagement.njk', data);
+        }
+      )
+      .catch(
+        err => {
+          console.warn("something went wrong:", err);
+          res.status(500).send(err);
+        }
+      );
   });
 
+  /*
+    router.post('/userManagement/:station', checkAuth, function (req, res) {
```

Comment:
Why is there a whole section of code commented out? If it is not used it should be deleted.